[Bug]:  Failed to calculate the value of task ':geolocator_android:processReleaseManifest' property 'packageOverride'.

### Please check the following before submitting a new issue.

- [X] I have searched the [existing issues](https://github.com/baseflow/flutter-geolocator/issues).
- [X] I have carefully [read the documentation](https://github.com/Baseflow/flutter-geolocator/blob/main/geolocator/README.md) and verified I have added the required platform specific configuration.

### Please select affected platform(s)

- [X] Android
- [X] iOS
- [ ] Linux
- [ ] macOS
- [ ] Web
- [ ] Windows

### Steps to reproduce

Using geolocator package to build an app with fluttermap.
Everythink works as web-emulator. But once I run the android build, I get the build error message.

### Expected results

-

### Actual results

FAILURE: Build failed with an exception.

* What went wrong:

Execution failed for task ':geolocator_android:processReleaseManifest'.

> Failed to calculate the value of task ':geolocator_android:processReleaseManifest' property 'packageOverride'.

> Failed to query the value of property 'applicationId'.

> Package Name not found in /var/workspace/.pub-cache/hosted/pub.dev/geolocator_android-4.2.1/android/src/main/AndroidManifest.xml

### Code sample

<details><summary>Code sample</summary>

```dart
[Paste your code here]
```

</details>


### Screenshots or video

<details>
<summary>Screenshots or video demonstration</summary>

[Upload media here]

</details>


### Version

9.0.2

### Flutter Doctor output

<details><summary>Doctor output</summary>

FAILURE: Build failed with an exception.

* What went wrong:

Execution failed for task ':geolocator_android:processReleaseManifest'.

> Failed to calculate the value of task ':geolocator_android:processReleaseManifest' property 'packageOverride'.

> Failed to query the value of property 'applicationId'.

> Package Name not found in /var/workspace/.pub-cache/hosted/pub.dev/geolocator_android-4.2.1/android/src/main/AndroidManifest.xml

</details>

--- a/geolocator_android/CHANGELOG.md
+++ b/geolocator_android/CHANGELOG.md
@@ -1,3 +1,7 @@
+## 4.2.2
+
+* Adds back the `applicationId` property of the AndroidManifest.xml file to keep backwardscompatibility with older applications that still rely on Gradle version <7.0.
+
 ## 4.2.1
 
 * Fixes a bug where the `stopForeground` method was called with the wrong constant, resulting in a build time error.

--- a/geolocator_android/android/src/main/AndroidManifest.xml
+++ b/geolocator_android/android/src/main/AndroidManifest.xml
@@ -1,4 +1,5 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="com.baseflow.geolocator">
 
     <application>
         <service

--- a/geolocator_android/pubspec.yaml
+++ b/geolocator_android/pubspec.yaml
@@ -2,7 +2,7 @@ name: geolocator_android
 description: Geolocation plugin for Flutter. This plugin provides the Android implementation for the geolocator.
 repository: https://github.com/baseflow/flutter-geolocator/tree/main/geolocator_android
 issue_tracker: https://github.com/baseflow/flutter-geolocator/issues?q=is%3Aissue+is%3Aopen
-version: 4.2.1
+version: 4.2.2
 
 environment:
   sdk: ">=2.15.0 <4.0.0"

